src/styles/blocks/card.scss

```
@@ -0,0 +1,57 @@
+.card {
+  width: $card-max-width;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color-light;
+  border-radius: 5px;
+  &__image {
+    display: flex;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+  &__title {
```

would be better to add empty lines between blocks
```suggestion
  border-radius: 5px;

  &__image {
    display: flex;
    margin: 0 auto;
    width: 160px;
    height: 134px;
  }

  &__title {
```